L1Trigger/L1THGCal/plugins/geometries/HGCalTriggerGeometryV9Imp3.cc

```diff
@@ -928,9 +913,9 @@ void HGCalTriggerGeometryV9Imp3::etaphiMappingFromSector0(int& ieta, int& iphi,
 }
 
 HGCalGeomRotation::WaferCentring HGCalTriggerGeometryV9Imp3::getWaferCentring(unsigned layer, int subdet) const {
-  if (subdet == ForwardSubdetector::HGCEE) {  // CE-E
+  if (subdet == HGCalTriggerSubdetector::HGCalEETrigger) {  // CE-E
     return HGCalGeomRotation::WaferCentring::WaferCentred;
-  } else if (subdet == ForwardSubdetector::HGCHEF || subdet == ForwardSubdetector::HGCHEB) {
+  } else if (subdet == HGCalTriggerSubdetector::HGCalHSiTrigger) {
```

Comment:
Is there a reason to have removed `HGCalHScTrigger`?

Reply:
The reason is that the type of WafterCentring isn't really appropriate for`HGCalHScTrigger` (only refers to silicon layers) - but maybe there's a better way to catch this?

Reply:
Maybe a warning message could be issued

Reply:
thanks, I've added one